Readme.md

```diff
@@ -61,29 +61,27 @@ Fetch the [postgres](https://hub.docker.com/_/postgres) docker image before runn
 docker pull postgres
 ```
 
+### Local Node
+
+You'll need to run a local node like geth. Start up a new chain and take note of the dev addresses.
+
```

Comment:
We can probably add or link some instructions on how to run a testnet.

https://book.getfoundry.sh/anvil/

Reply:
Done!